python/taichi/main.py

```diff
@@ -88,12 +88,42 @@ def __call__(self):
             # TODO: do we really need this?
             if args.command.endswith(".py"):
                 TaichiMain._exec_python_file(args.command)
-            print(f"{args.command} is not a valid command!")
-            self.main_parser.print_help()
+            else:
+                print(f"{args.command} is not a valid command!")
+                self.help()
             return 1
 
         return getattr(self, args.command)(sys.argv[2:])
 
+    @register
+    def help(self, arguments: list = sys.argv[2:]):
+        """Show help message."""
+        parser = argparse.ArgumentParser(prog='ti help',
+                                         description=f"{self.help.__doc__}")
+        parser.add_argument(
+            '-s',
+            '--save',
+            required=False,
+            dest='save',
+            action='store_true',
+            help="Save source code to current directory instead of running it")
```

Comment:
this seems to be a mis copy-paste, `ti help` doesn't need any argument if I understand it correctly.